### PR TITLE
Cleanup R-6 and add an integrative problem

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

**Comment by ethanwhite** [comment]
@brymz add changes to the solutions for these as soon as you get the chance. Thanks.

**Comment by ethanwhite** [comment]
I just pushed a PR to the solutions repo for Basics-R-2.

File: exercises/Making-choices-2-R.md [hunk 1]
```diff
@@ -11,8 +11,7 @@ return `FALSE`. Two points are considered near to each other if the absolute
 value of the difference in their latitudes is less than one and the absolute
 value of the difference in their longitudes is less than one.
 
-Fill in the `_________` in the function to make it work and then use it to check 
-if the following pairs of points are near or not and print out the answers.
+Fill in the `_________` in the function to make it work.
```
Comment (brymz) [inline exercises/Making-choices-2-R.md]: Should this be task 1?

File: exercises/Making-choices-2-R.md [hunk 2]
```diff
@@ -26,9 +25,18 @@ near <- function(lat1, long1, lat2, long2){
...
-longitude = -111.83.
-2. Point 1: latitude = 29.65, longitude = -82.33. Point 2: latitude = 30.5,
-longitude = -82.8.
-3. Point 1: latitude = 48.86, longitude = 2.35. Point 2: latitude = 41.89,
-longitude = 2.5.
+1. Improve the documentation for the function so that it is clear what near
+   means and what output the user should expect.
+2. Check if Point 1 is near: latitude = 29.65, longitude = -82.33. Point 2:
+   latitude = 41.74, longitude = -111.83.
+3. Check if Point 2 is near: latitude = 29.65, longitude = -82.33. Point 2: latitude = 30.5,
```
Comment (brymz) [inline exercises/Making-choices-2-R.md]: Check if Point 2 is near... Point 2? Something similar happens in task 6 also.

File: exercises/Making-choices-2-R.md (hunk 1 above)
Comment (ethanwhite) [inline exercises/Making-choices-2-R.md]: Yes, but folks are already working on this so it's too late to change it.

**Comment by ethanwhite** [comment]
> Should this be task 1?

Yes

File: exercises/Making-choices-2-R.md (hunk 2 above)
Comment (brymz) [inline exercises/Making-choices-2-R.md]: #258

**Pull request merged**